requirements.txt

```diff
@@ -8,8 +8,8 @@ requests-oauthlib
 boto3
 elasticsearch-dsl>=5.0.0,<6.0.0
 aws-requests-auth
-django==1.11
-django-cas-ng
+django==2.0.1
+git+git://github.com/mingchen/django-cas-ng.git#egg=django-case-ng
```

Comment:
Will be good to lock to some specific commit hash because changes in **django-cas-ng** can bring the app to a broken status. You also have a typo in the egg: `django-case-ng` should be `django-cas-ng` or PIP will show a warning. Also looks like the master branch doesn't have full Django 2.0 support yet, somebody opened a PR recently on https://github.com/mingchen/django-cas-ng/pull/150 and it's passing so this line will probably look like:

```
git+git://github.com/wj/django-cas-ng.git@39b751ea05110ba01f0f37ce698d72e1ab1f537a#egg=django-cas-ng
```
Other than this the rest of the PR looks good to me :)

Reply:
You can leave a `FIXME` comment to use the released version or master once the PR is merged there.

Reply:
The goal is to merge the branch when version v3.5.9 is released at Pypi as the latest version. Until that, this branch will give an idea of upstream migrations. I believe I have already pointed out this point in PR. Many thanks for your replay :)

Reply:
Perfect, rest of your code looks great, love the new style URLs!